### ISSUE TITLE
Allow disabling or caching of contig metrics calculation

### ISSUE THREAD

**Issue by ctb** [Blahah/transrate#166]
Calculating the contig metrics takes some time. Especially when debugging, this is annoying.  Should be able to do this just once, n'est ce pas?

**Comment by Blahah** [comment]
Yes, we should. I think perhaps take a hash of the assembly and store it, along with the metrics, in an `#{assembly_name}.json`. On subsequent runs, check the hash for an assembly with that name matches the stored one, and if so reuse the metrics.
